Components/SoarDebug/DebugTask.cpp

```diff
@@ -158,6 +165,14 @@ void DebugTask::HandleDebugMessage(const char* msg)
         Command cmd2(REQUEST_COMMAND, IMU_REQUEST_DEBUG);
         IMUTask::Inst().GetEventQueue()->Send(cmd2);
     }
+    else if (strcmp(msg, "radiohb") == 0) {
```

Comment:
Let's use the one-liner `WatchdogTask::Inst().SendCommand(Command(BLAH, BLAH));`, its a bit cleaner, we should eventually update all simple sends like this to use the SendCommand interface.